lib/constants.js

```diff
@@ -18,13 +19,19 @@ export const ECDSA_2019_SUITE_CONTEXT_V1_URL =
 export const MULTIKEY_CONTEXT_V1_URL = 'https://w3id.org/security/multikey/v1';
 export const MULTIBASE_BASE58_HEADER = 'z';
 
+// Multicodec secp256k1-pub header (0xe701)
+// https://github.com/public-square/jwk-multibase-key-converter-php/blob/main/src/MultibaseConversionFactory.php#L17
```

Comment:
We can just put the value and its varint expression from the table here (to be consistent with the others below):

https://github.com/multiformats/multicodec/blob/master/table.csv#L93

```suggestion
// Multicodec secp256k1-pub header (0xe7 varint -> 0xe701 hex)
```

Reply:
I felt documenting where to find this information was useful for future additions. Not sure if many will occur.